ui/src/components/Sidebar/GroupList.tsx

```diff
@@ -63,7 +63,7 @@ export default function GroupList({
       computeItemKey={(_i, [flag]) => flag}
       itemContent={itemContent}
       components={components}
-      className="h-full w-full overflow-x-hidden"
+      className="h-full w-full space-y-0.5 overflow-x-hidden"
```

Comment:
@patosullivan @arthyn I'm attempting to apply this inter-element styling I applied to the channel list in this PR, but it doesn't appear to function/work as expected — any idea what might be going on here?

Reply:
I'm specifically'referring to adding `space-y-0.5` to the ClassName here

Reply:
It's because that utility is intended to be used with flex, but we aren't using flex here. You could add it to the classes on the Wrapper component within the SidebarItem component. I'll push a pr to show what I mean.

Reply:
I also included an update to the pre-commit hook that will update the snapshots before commit so you don't see these build errors in CI. Something we should have done a long time ago tbh.